server/uno_test.go

```diff
@@ -187,28 +193,32 @@ func TestCheckForCardInHand(t *testing.T){
 }
 
 func TestCreatePlayer(t *testing.T){
+	os.Setenv("DB_TYPE", "MOCK")
 	// get the database
 	database, err := db.GetDb()
 	assert.Nil(t, err, "could not find database")
 	// use the createPlayer function
-	player, err := createPlayer("test")
-	assert.Nil(t, err, "could not create player")
+	player, err := createPlayer("test") 
+ 	assert.Nil(t, err, "could not create player")
 	// Lookup the player in the database to see if it is there
 	databasePlayer, err := database.LookupPlayer(player.ID)
 	assert.Nil(t, err, "could not find player")
 	// Test to see if the database player and the created player are the same
-	assert.Equal(t, player, databasePlayer)
+	assert.Equal(t, player, databasePlayer) 
 }
 
 func TestJoinGame(t *testing.T){
+	os.Setenv("DB_TYPE", "MOCK")
```

Comment:
Does this have to be done in each test or should this instead be done from the command line test call?

Reply:
Hmm sounds like a better way, I am not sure how to actually go about doing that, do I set the environment variable in the same command as go test -cover? Or would I do an export DB_TYPE="MOCK" prior to running my tests?